### ISSUE TITLE
HTML encoded strings are decoded

### ISSUE THREAD

**Issue by doits** [nono/HTML-Truncator#14]
Just stumbled at this:

```
HTML_Truncator.truncate('12345678901', 10, length_in_chars: true)
=> "1234567890…" # good

HTML_Truncator.truncate('<br>12345678901', 10, length_in_chars: true)
=> "<br>1234567890…" # good

HTML_Truncator.truncate('<br>&lt;br&gt;12345678901', 10, length_in_chars: true)
=> "<br><br>123456…" # bad, second '<br>' is decoded!

HTML_Truncator.truncate('&lt;br&gt;', 10, length_in_chars: true)
=> "&lt;br&gt;" # inconsistent: if length is shorter, it is not decoded as opposed to example before
```

I think the method should never decode strings. The encoded chars could count as 1 length (`&lt;` is length 1 etc), so:

```
HTML_Truncator.truncate('<br>&lt;br&gt;12345678901', 10, length_in_chars: true)
=> "<br>&lt;br&gt;123456…"
```

This would make most sense I think.

**Comment by nono** [comment]
Hello,

thanks for your bug report. I fixed it in 66abb5b087471c0775f3c99bcb816e8e40ad3e47 and release a new version with it (0.4.1). Is it OK for you?

**Comment by doits** [comment]
Thanks, looks good on first sight and tests
